src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -0,0 +1,19 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+
+<Project>
+ 
+  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)"/>
+  
+  <Target Name="RunPackageValidation" AfterTargets="Pack">
+    <PropertyGroup>
+      <PackagePath Condition="'$(PackagePath)' == ''">$(PackageOutputAbsolutePath)$(PackageId).$(PackageVersion).nupkg</PackagePath>
+    </PropertyGroup>
+
+    <ValidatePackage PackagePath="$(PackagePath)"
+                     RuntimeGraph="$(RuntimeGraph)"
+                     NoWarn="$(NoWarn)"
+                     RunApiCompat="$([MSBuild]::ValueOrDefault('$(RunApiCompat)', 'false'))"
```

Comment:
I think this is unnecessary, an unset bool parameter should be defaulted to false.

Reply:
I thought that we wanted to run api compat by default as part of package validation? 

The only one that should be opt-in is running against the previous package version as that one needs to download that package.